docs/kubefirst/local/explore/user-creation.md

```diff
@@ -0,0 +1,63 @@
+## Atlantis Example / User Creation walkthrough
+
+In this tutorial we will show how to add users to your local cluster through [Atlantis ](https://www.runatlantis.io/), which will allow a preview of how changes made will be expressed through terraform before branches are merged into your repository.
+
+Navigate to the `gitops` repository in your personal GitHub, clone the contents, and create a new branch:
+
+```
+cd gitops
+git checkout -b new-user
+```
+
+The file `terraform/users/admins-github.tf` contains blocks that represent admin users - the kubefirst_bot user, and a commented-out admin_one user:
+
+```
+module "admin_one" {
+  source = "./modules/user/github"
+  acl_policies        = ["admin"]
+  email               = "admin@your-company-io.com"
+  first_name          = "Admin"
+  github_username     = "admin_one_github_username"
+  last_name           = "One"
+  initial_password    = var.initial_password
+  username            = "aone"
+  user_disabled       = false
+  userpass_accessor   = data.vault_auth_backend.userpass.accessor
+}
+```
+
+Uncomment and edit this code to replace the values for the email, first_name, github_username, last_name, and username before pushing to your branch. 
+Note: If you are doing using this walkthrough simply to test Atlantis, you do not need to update these fields to be accurate.
+
+```
+git add .
+git commit -m feat: add new user
+git push --set-upstream origin new-user
+```
+
+Create a merge request. This will kick off the Atlantis workflow. Within a minute or so of submitting the merge request, a comment will appear on the merge request that shows the terraform plan with the changes it will be making to your infrastructure. 
```

Comment:
```suggestion
Create a pull request. This will kick off the Atlantis workflow. Within a minute or so of submitting the pull request, a comment will appear on the pull request that shows the terraform plan with the changes it will be making to your infrastructure. 
```